packages/component/src/SendBox/ConnectivityStatus.js

```diff
@@ -62,42 +63,77 @@ const connectConnectivityStatus = (...selectors) =>
   connectToWebChat(({ connectivityStatus, language }) => ({ connectivityStatus, language }), ...selectors);
 
 const ConnectivityStatus = ({ connectivityStatus, language, styleSet }) => (
-  <div aria-live="polite" role="status">
+  <div aria-atomic="false" aria-live="polite" role="status">
     <DebouncedConnectivityStatus
       interval={
         connectivityStatus === 'uninitialized' || connectivityStatus === 'error' ? 0 : CONNECTIVITY_STATUS_DEBOUNCE
       }
     >
-      {() =>
-        connectivityStatus === 'connectingslow' ? (
-          <div className={styleSet.warningNotification}>
-            <WarningNotificationIcon />
-            {localize('SLOW_CONNECTION_NOTIFICATION', language)}
-          </div>
-        ) : connectivityStatus === 'error' || connectivityStatus === 'notconnected' ? (
-          <div className={styleSet.errorNotification}>
-            <ErrorNotificationIcon />
-            {localize('FAILED_CONNECTION_NOTIFICATION', language)}
-          </div>
-        ) : connectivityStatus === 'uninitialized' ? (
-          <div className={styleSet.connectivityNotification}>
-            <SpinnerAnimation />
-            {localize('INITIAL_CONNECTION_NOTIFICATION', language)}
-          </div>
-        ) : connectivityStatus === 'reconnecting' ? (
-          <div className={styleSet.connectivityNotification}>
-            <SpinnerAnimation />
-            {localize('INTERRUPTED_CONNECTION_NOTIFICATION', language)}
-          </div>
-        ) : connectivityStatus === 'sagaerror' ? (
-          <div className={styleSet.errorNotification}>
-            <ErrorNotificationIcon />
-            {localize('RENDER_ERROR_NOTIFICATION', language)}
-          </div>
-        ) : (
-          connectivityStatus === 'reconnected' || (connectivityStatus === 'connected' && false)
-        )
-      }
+      {() => {
+        if (connectivityStatus === 'connectingslow') {
```

Comment:
For readability, could you turn this into a `switch` statement?

Reply:
Question, linting error eslint(no-case-declarations) comes up if I move to switch:
![image](https://user-images.githubusercontent.com/14900841/62581258-7e2f2900-b85d-11e9-8cb0-bb798df6809d.png)


Do you still prefer switch statement with this rule disabled at the top of the file?

Reply:
lol nvm it needed braces

Reply:
Oh yea, sorry missed that.

Hate `{}` in switch-case. If you are okay, let's stay with if-then for now.